spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/properties/ComposedTaskProperties.java

```diff
@@ -191,6 +191,12 @@
 	 */
 	private boolean skipTlsCertificateVerification = false;
 
+	/**
+	 * Establish the transaction isolation level for the Composed Task Runner.
+	 * Default is ISOLATION_REPEATABLE_READ.
+	 */
+	private String transactionIsolationLevel = "ISOLATION_REPEATABLE_READ";
```

Comment:
Was going to nitpick that this is a string but looks like factory in batch is also using string. That was strange as in spring transaction classes these are enums. I would be nice to give a set of valid options but not sure how feasible that is.

Reply:
It's worth to check if boot has any similar configuration options and if so then use same type.

Reply:
I spelunked boot, but didn't find anything that mapped a string version of transaction isolation level to an enumeration.

Reply:
I found https://github.com/spring-projects/spring-boot/blob/3e21ae12dc2653ab7ce7d561c29bc018a57a4103/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/batch/JpaBatchConfigurer.java#L58 and that is just string. We can then use same String type even if enum would be better. At least it's same in both places.